googlemaps/addressvalidation.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright 2014 Google Inc. All rights reserved.
```

Comment:
Change license header to 2022

Reply:
I will add this change

Note: I will submit a new pull request as single commit for all changes to this review round